src/ArduinoIoTCloud.h

```diff
@@ -134,6 +143,10 @@ class ArduinoIoTCloudClass {
   ConnectionManager *connection;
   static void onMessage(int length);
   void handleMessage(int length);
+  ArduinoIoTSynchronizationStatus syncStatus = SYNC_STATUS_SYNCHRONIZED;
```

Comment:
I'd suggest to prefix `syncStatus` with an underscore `_` like all other member variables.
```suggestion
  SyncStatus _syncStatus = SyncStatus::Synchronized;
```

Reply:
Done